gamemodes/terrortown/gamemode/shared/events/bodyfound.lua

```diff
@@ -0,0 +1,62 @@
+--- @ignore
+
+if CLIENT then
+	EVENT.icon = Material("")
```

Comment:
Why is the Icon initialized with `Material("")` instead of nil?

Reply:
has no reason as the icon will be added in the UI PR, I can set it to nil if you so prefer.

Reply:
nil would be better, as we dont need an unused material object